.travis/before_install-linux.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -ev
+
+sudo apt-get install -qq pkg-config fuse
+sudo modprobe fuse
+sudo chmod 666 /dev/fuse
+sudo chown root:$USER /etc/fuse.conf
```

Comment:
test times jumped up by 10min in linux and 3min in osx. (they all used to run ~4min (when they dont get stalled))